src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,26 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Transaction transaction = null;
+        try (Session session = factory.openSession()) {
+            transaction = session.beginTransaction();
+            session.persist(movie);
```

Comment:
Consider using `session.save(movie)` instead of `session.persist(movie)` if you want to ensure that the entity is immediately saved and the generated identifier is available. `persist` is typically used for entities that are not yet persisted and does not guarantee the immediate generation of an identifier.